client/src/components/MultiJurisdictionAudit/CSVForm/index.tsx

```diff
@@ -131,43 +140,35 @@ const CSVFileForm = ({
                         .
                       </SuccessLabel>
                     )}
-                </>
-              )}
-            </FormSection>
-            <div>
-              {isEditing ? (
-                <FormButton
-                  type="submit"
-                  intent="primary"
-                  onClick={handleSubmit}
-                  loading={isSubmitting || isProcessing}
-                  disabled={!enabled}
-                >
-                  Upload File
-                </FormButton>
-              ) : (
-                // We give these buttons a key to make sure React doesn't reuse
-                // the submit button for one of them.
-                <>
+                </FormSection>
+                <div>
+                  {/* We give these buttons a key to make sure React doesnt
```

Comment:
"doesn't"

Reply:
lol i had to take out the apostrophe because the linter wanted me to escape it